plugins/pulp_python/plugins/importers/importer.py

```diff
@@ -65,11 +66,12 @@ def import_units(self, source_repo, dest_repo, import_conduit, config, units=Non
         :rtype:                list
         """
         if units is None:
-            search = criteria.UnitAssociationCriteria(type_ids=[constants.PACKAGE_TYPE_ID])
-            units = import_conduit.get_source_units(criteria=search)
+            units = chain(*repo_controller.get_unit_model_querysets(source_repo.repo_obj.repo_id,
+                                                                    models.Package))
 
+        units = list(units)
         for u in units:
-            import_conduit.associate_unit(u)
+            repo_controller.associate_single_unit(dest_repo.repo_obj, u)
```

Comment:
Where is the call to `rebuild_content_unit_counts()` for this single unit association? The old style `import_conduit.associate_unit()` does update the repo content unit count but with the mongoengine based methods `rebuild_content_unit_counts()` needs to be explicitly called.